clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -177,10 +177,14 @@ CIRGenModule::CIRGenModule(mlir::MLIRContext &context,
     sob = sob::SignedOverflowBehavior::trapping;
     break;
   }
+
+  // FIXME(cir): Implement a custom CIR Module Op and attributes to leverage
+  // MLIR features.
```

Comment:
Reminder: check if there is already an issue for this.

Reply:
There you go: https://github.com/llvm/clangir/issues/644